src/pages/[platform]/prev/build-a-backend/auth/admin_actions/index.mdx

```diff
@@ -0,0 +1,369 @@
+import { getCustomStaticPath } from '@/utils/getCustomStaticPath';
+                  
+export const meta = {
+  title: 'Set up admin actions',
+  description: 'Learn how to expose administrative actions for your Cognito User Pool to your end user applications.',
+  platforms: [
+    'flutter',
+  ],
+  canonicalObjects: [
+    {
+      platforms: [
+        'flutter',
+      ],
+      canonicalPath: '/javascript/build-a-backend/auth/admin-actions/'
+    }
+  ]
+};
+
+export const getStaticPaths = async () => {
+  return getCustomStaticPath(meta.platforms);
+};
+
+export function getStaticProps(context) {
```

Comment:
this page should exist under "gen1" right?

Reply:
The page exists under gen1. The folder name had a typo in it, it should've been `admin-actions` not `admin_actions`. I just corrected.